R/barmulticolor.R

```diff
@@ -192,27 +190,40 @@ BarMultiColor <- function(x,
     margins <- setMarginsForLegend(margins, FALSE, NULL, NULL)
     margins <- setCustomMargins(margins, margin.top, margin.bottom, margin.left,
                     margin.right, margin.inner.pad)
+    margins$autoexpand <- margin.autoexpand
 
     x <- axisFormat$labels
     y <- as.numeric(chart.matrix[,1])
+    y.filled <- ifelse(is.finite(y), y, 0)
     x.text <- formatByD3(y, x.hovertext.format)
     marker = list(color = toRGB(colors, alpha = opacity),
                 line = list(color = toRGB(marker.border.colors,
                 alpha = marker.border.opacity), width = marker.border.width))
     hoverfont <- list(color = autoFontColor(colors), size = hovertext.font.size,
                 family = hovertext.font.family)
 
+    # Add invisible trace to force all labels to be shown 
+    # (including missing) 
+    tmp.min <- if (any(is.finite(chart.matrix))) min(chart.matrix, na.rm = TRUE)
```

Comment:
Same as above (`-Inf` comment)

Reply:
Fixed in the same way as the comment above